charts/hnc/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Caution: Don't change namespaceOverride from hnc-system.
```

Comment:
Is there a reason we don't just hardcode these in `_helpers.tpl` since the aren't supposed to be overriden?

Reply:
@rjbez17 
As you say, I think it good to hardcode them.
So I've fixed that.